src/components/ui/Textarea.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components/macro'
+
+const StyledTextarea = styled.textarea`
+  border: 1px solid ${({ theme }) => theme.secondaryBackground};
+  box-sizing: border-box;
+  border-radius: 12px;
+  padding: 15px 20px;
+  resize: none;
```

Comment:
small aside: some power users do really like resize:vertical which is why if it doesn't mess up styling, i think that might be good (https://catalin.red/css-resize-none-is-bad-for-ux/)

Reply:
@SirajChokshi

Reply:
We can use `resize: vertical`!